ground/src/main/java/com/google/android/ground/ui/editsubmission/EditSubmissionFragment.java

```diff
@@ -314,7 +314,7 @@ private void initPhotoTask(PhotoTaskViewModel photoTaskViewModel) {
 
   private void observeSelectPhotoClicks(PhotoTaskViewModel fieldViewModel) {
     fieldViewModel
-        .getShowDialogClicks()
+        .getTakePhotoClicks()
```

Comment:
Some comment as above here. I realize this was submitted as part of a previous PR, but if it's easy let's change it now so we don't forget?

Reply:
I think the EditSubmissionFragment will likely be replaced since it's a holdover from the previous prototype so I think maybe we can just leave it as is.

Thoughts?